yb-voyager/cmd/importData.go

```diff
@@ -933,12 +933,19 @@ func executeSqlFile(file string) {
 
 	var errOccured = 0
 	sqlStrArray := createSqlStrArray(file, "")
-	for _, sqlStr := range sqlStrArray {
+	retryCount := INDEX_RETRY_COUNT
+	for i := 0; i < len(sqlStrArray); {
+		sqlStr := sqlStrArray[i]
 		log.Infof("Run query %q on target %q", sqlStr[1], target.Host)
 		_, err := conn.Exec(context.Background(), sqlStr[0])
 		if err != nil {
-			log.Errorf("Run query %q on target %q: %s", sqlStr[1], target.Host, err)
-			if strings.Contains(err.Error(), "already exists") {
+			log.Errorf("Index creation failed %s", err)
```

Comment:
Don't assume that this function is called only for "index creation". This function can also be called for sequence creation:
```
	if utils.FileOrFolderExists(sequenceFilePath) {
		fmt.Printf("setting resume value for sequences %10s", "")
		go utils.Wait("done\n", "")
		executeSqlFile(sequenceFilePath)
	}
```